### DISCUSSION TITLE
Copilot inline tray resets VS Code zoom level

### DISCUSSION THREAD

**Discussion by sselin-co** [community/community#7833] [Copilot Conversations] [answered]
First off, I adore the plugin!

When working with a custom window zoom level in VS Code (`Ctrl + '+'/'-'`), using the inline pop-up button tray for suggestions resets the zoom level to seemingly 100%. This happens regardless of which button in the tray is clicked. This does not happen when using Tab to auto-fill suggestions or using suggestions from the Copilot suggestions window. Nothing relating to window zoom shows up in the Copilot logs output. Unfortunately, I don't have another computer to test if this is specific to my machine. 

*Pre click*
![image](https://user-images.githubusercontent.com/39775792/142245584-d7c31937-727c-470f-8acf-7e09884e8a5b.png)
*Post click*
![image](https://user-images.githubusercontent.com/39775792/142245488-2227c028-4272-4a0c-9b56-d64725e41a71.png)

Any help would be appreciated since I frequently use custom zoom levels and the button tray. Thank you!

**Comment by Urethramancer** [comment]
I always have VSCode zoomed in a few notches, but have never run into that. Is there any particular language it happens in, or is it everything? Could there be an old theme or something getting in the way?

**Reply by sselin-co** [reply]
It seems to be everything, happens with my node and django projects in all associated file types. I can try disabling any themes and UI extensions then incrementally add them back to see if something strange is conflicting.

**Comment by Krzysztof-Cieslak** [comment, marked as answer]
Hey, thanks for the report. I've notified the VSCode Team about this issue, you can track it here: https://github.com/microsoft/vscode/issues/137512

**Reply by sselin-co** [reply]
Thanks so much! The issue has since been resolved 😄
